docs/reference/permissions.md

```diff
@@ -0,0 +1 @@
+::: synapseclient.models.permission
```

Comment:
I committed 1 additional change here - Please review and let me know your thoughts: https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1037/commits/315eb98b66a896a6ae1d4dc28b512c0aa7bd2638

The reason I added it is because we have a docstring on each of the attributes, and we have them in the class docstring it ends up duplicating the information in the doc pages. See the screenshots below. Unfortunately because they serve 2 different purposes, [See this slack thread](https://sagebionetworks.slack.com/archives/C02M7QR8D46/p1702070318432449), they cannot be combined at the moment.

[In this settings](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/mkdocs.yml#L107) I have it setup to not show if no docstring, but again, since we are including it the attributes show up twice.

Before:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/2cc0e36e-7f41-4cfd-86bc-43571f9dc254)

After:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/e26ce429-7585-4260-8b7c-a4f6e9a58b46)

Reply:
I agree that we don't want to show duplicate information on the doc. I like your idea and found `show_docstring_attributes` might also be helpful. Let me test it and circle back.

Reply:
@BryanFauble Here is the output of using show_docstring_attributes as False. 
1. If we add show_docstring_attributes as false, nothing except the class description will be shown.
<img width="751" alt="Screenshot 2023-12-29 at 11 12 11 AM" src="https://github.com/Sage-Bionetworks/synapsePythonClient/assets/90745557/2b550ebb-238f-42ed-bc88-a050b1fb4866">
2. If we comment out the member section, we get the attribute and class property (access_types) as paragraph
<img width="764" alt="Screenshot 2023-12-29 at 11 13 43 AM" src="https://github.com/Sage-Bionetworks/synapsePythonClient/assets/90745557/d5bfd8fd-3178-4ee9-acf1-49fa0a556006">

<img width="737" alt="Screenshot 2023-12-29 at 11 13 59 AM" src="https://github.com/Sage-Bionetworks/synapsePythonClient/assets/90745557/6bfdb46e-082a-40e3-9c4f-1fac8edd2d41">


I will go with the approach you commited since it will keep the attribute table. 


I will accept your commit si